src/App.js

```diff
@@ -1,33 +1,32 @@
 import React, { Component } from 'react';
 import './App.scss';
+import { SearchMovie } from './components/SearchMovie/SearchMovie';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  filterMovies = (event) => {
+    this.setState({ query: event.target.value });
+  };
```

Comment:
let's pass here event.target.value as an argument

```suggestion
  saveQuery = (value) => {
    this.setState({ query: value });
  };
```

Reply:
also, let's save lowercased value